test/remote/gateways/remote_braintree_blue_test.rb

```diff
@@ -103,6 +103,14 @@ def test_successful_setup_purchase
     assert_not_nil response.params['client_token']
   end
 
+  def test_successful_setup_purchase_with_merchant_account_id
+    assert response = @gateway.setup_purchase(merchant_account_id: fixtures(:braintree_blue)[:merchant_account_id])
+    assert_success response
+    assert_equal 'Client token created', response.message
+
+    assert_not_nil response.params['client_token']
```

Comment:
Are you able to assert that a field in the response matches the `merchant_account_id` provided?